src/entities/communitySbt.ts

```diff
@@ -18,7 +18,8 @@ export type SBTConstructorArgs = {
     id: string;
     signer: Signer| null;
     coingeckoKey?: string;
```

Comment:
What I'd really suggest is to make these fields non-optional. They need to be passed by the UI. Hence, we improve readability a lot by avoiding these checks ``if (!field) { return; } ...``

Reply:
Let's align on this during the meeting next Thursday